src/App/Header/LanguageSwitch.tsx

```diff
@@ -80,6 +81,10 @@ const lngDetails: Record<Language, { translation: TFunctionKeys; icon: React.Rea
     translation: 'lng.chineseTraditional',
     icon: <Taiwan size="md" />,
   },
+  'fi-FI': {
+    translation: 'lng.finnish',
+    icon: <Finnish size="md" />,
```

Comment:
You would need to export here `src/components/Icons/index.ts` otherwise fails on build